salem/datasets.py

```diff
@@ -497,18 +500,19 @@ def __init__(self, center_ll=(11.38, 47.26), size_x=640, size_y=640,
           any keyword accepted by motionless.CenterMap (e.g. `key` for the API)
         """
 
-        if 'scale' in kwargs:
-            raise NotImplementedError('scale not supported')
+        if scale < 0 or scale > 2 or type(scale) is not int:
```

Comment:
I don't think we need to check for the input to google API in salem. We don't do it for the other parameters either: the salem part of the code (the map projection) will work for whatever value of the scale, and we shouldn't care if users misuse the google API.